cmd/main.go

```diff
@@ -27,6 +29,23 @@ func main() {
 	if err != nil {
 		panic(err)
 	}
+
+	var programLevel = new(slog.LevelVar)
+
+	switch cfg.LogLevel {
+	case "info":
+		programLevel.Set(slog.LevelInfo)
+	case "warn":
+		programLevel.Set(slog.LevelWarn)
+	case "error":
+		programLevel.Set(slog.LevelError)
```

Comment:
[nit] use generated values from package `loglevel`

```suggestion
	switch cfg.LogLevel {
	case loglevel.Info:
		programLevel.Set(slog.LevelInfo)
	case loglevel.Warn:
		programLevel.Set(slog.LevelWarn)
	case loglevel.Error:
		programLevel.Set(slog.LevelError)
```